Gopkg.lock

```diff
@@ -254,6 +254,14 @@
   revision = "20f1fb78b0740ba8c3cb143a61e86ba5c8669768"
   version = "v0.5.0"
 
+[[projects]]
+  branch = "master"
+  digest = "1:b669bf7ea147e4d75e097e89f45c1e7de52e79dc913bbbae94b79177f7b075b6"
+  name = "github.com/heptiolabs/healthcheck"
```

Comment:
hi, thanks for contributing this feature! because this is a k8s project, for consistent behavior, what about using this package instead? `https://godoc.org/k8s.io/apiserver/pkg/server/healthz`